src/core/reopt.jl

```diff
@@ -530,6 +537,10 @@ function add_variables!(m::JuMP.AbstractModel, p::REoptInputs)
         end
     end
 
+    if !isempty(p.techs.steam_turbine)
+        @variable(m, dvThermalToSteamTurbine[p.techs.all_for_steam_turbine, p.time_steps] >= 0)
```

Comment:
I forget the exact reason I did this in the API, but we'd have to change this line into two separate lines: https://github.com/NREL/REopt.jl/blob/c740b0797dc2ff97ef74c0b2aa0718113e3e87b3/src/constraints/load_balance.jl#L140

To only subtract `dvThermalToSteamTurbine` across `p.techs.can_supply_steam_turbine` and the new line would just be for `dvThermalProduction` across the set difference of `p.techs.all_for_steam_turbine` - `p.techs.can_supply_steam_turbine`. 

Otherwise, I agree we should avoid creating dV's if we don't need to, and the adding a line to the load balance is a very minor downside. I'm currently forcing that set-difference dV to zero. I just forget if there was a real substantial reason I did this, for some other place.

Reply:
Yep, that's what I was thinking too.  Using a set difference in the constraints is the preferred method (and matched the math), and I didn't see the DV `dvThermalToSteamTurbine` used anywhere for the larger set `p.techs.all_for_steam_turbine`.  But let me know if I just missed something

Reply:
Addressed in 60c288e674ee2ce218dc81953e5477f90044e077.